src/Raven.Server/Documents/Patch/BlittableObjectInstance.cs

```diff
@@ -142,7 +142,19 @@ public BlittableObjectProperty(BlittableObjectInstance parent, string property)
                                         value = string.Empty;
                                         break;
                                 }
-                                _value = value;
+                                
+                                if (long.TryParse(value, out var valueAsLong))
```

Comment:
I think we should just first do double, then long.
This is because JS have doubles,not longs

Reply:
Removed long since JavaScript numbers are always stored as double precision floating point.